app/controllers/aboutfranchisesorlettings/CheckYourAnswersAboutFranchiseOrLettingsController.scala

```diff
@@ -132,6 +132,7 @@ class CheckYourAnswersAboutFranchiseOrLettingsController @Inject() (
             logger.warn(s"Back link for premises license page reached with unknown enforcement taken value")
             controllers.routes.TaskListController.show().url
         }
+      case _                                   => controllers.routes.TaskListController.show().url
```

Comment:
Michal. Can we also include a log message (warn) so we know we reached a default case and have sent the user back to the task list. Thanks.